### ISSUE TITLE
improve accessibility of limel-tooltip by displaying it when receiving keyboard focus

### ISSUE THREAD

**Issue by Kiarokh** [Lundalogik/lime-elements#1858]
## Current behavior
The tooltip becomes visible in response to a mouse hover. 
But unfortunately not after the owning element receives keyboard focus, currently.

## Expected behavior
The tip should become visible when owning element receives keyboard focus (in addition to hover). To make the component keyboard accessible.

Read more: https://www.w3.org/TR/wai-aria-1.1/#tooltip

## Environment

- **lime-elements** version: 
- Framework used: 
- Logs:

**Comment by Kiarokh** [comment]
@reviewer you may be interested in reading this too:

https://github.com/Lundalogik/crm-feature/issues/3502#issuecomment-2037258158
